pkg/utils/hwaddr/hwaddr.go

```diff
@@ -20,37 +20,39 @@ import (
 )
 
 const (
-	ipRelevantByteLen      = 4
-	PrivateMACPrefixString = "0a:58"
+	ipRelevantByteLen       = 4
+	PrivateMACPrefixString  = "0a:58"
+	PrivateMACPrefixString6 = "6a:58"
 )
 
 var (
 	// private mac prefix safe to use
-	PrivateMACPrefix = []byte{0x0a, 0x58}
+	PrivateMACPrefix  = []byte{0x0a, 0x58}
+	PrivateMACPrefix6 = []byte{0x6a, 0x58}
 )
 
-type SupportIp4OnlyErr struct{ msg string }
+type InvalidIP4Err struct{ msg string }
 
```

Comment:
Should the name of line 34 and 36 be general for v4 and v6 or should a separate type and func be added for v6?